Update governance and community documents to reflect current status (#4049)
the-turing-way/the-turing-way: GOVERNANCE_ROLES.md

Information from these base files are also published as the [Contributors Record](https://book.the-turing-way.org/afterword/contributors-record) in the afterwards of the book.

Please refer to the Foreword of the book to learn about different kinds of [community roles](https://book.the-turing-way.org/foreword/community).

## Maintenance Level Roles

The "Maintenance" (“Collective choice") level participation allow different groups of people to come together to work on and promote specific area/initiative in the project as well as define/change process to make operational level participation easier.
Please read [Ways of Working document](https://github.com/the-turing-way/the-turing-way/blob/main/ways_of_working.md) for community structure, commitments and resources used by members at maintenance level.

Members participating in different Working Groups and projects carried out in _The Turing Way_ are listed below.

| Name | Position;  Affiliation | Community Role(s) | Role length | Previous roles in _The Turing Way_ |
|---|---|---|---|---|

> | Kirstie Whitaker <br>([@KirstieJane](https://github.com/KirstieJane)) | Executive Director, BIDS | JupyterHub -_The Turing Way_ Project co-lead (in her roles as the Director of BIDS), Founder | Nov 2024 - present | Project Lead of _The Turing Way_, Programme Director of Tools, Practices and Systems (TPS); The Alan Turing Institute, Oct 2018 - Nov 2024 |

Reviewer: ```suggestion
| Kirstie Whitaker <br>([@KirstieJane](https://github.com/KirstieJane)) | Executive Director, Berkeley Institute for Data Science (BIDS) | JupyterHub -_The Turing Way_ Project co-lead (in her roles as the Director of BIDS), Founder | Nov 2024 - present | Project Lead of _The Turing Way_, Programme Director of Tools, Practices and Systems (TPS); The Alan Turing Institute, Oct 2018 - Nov 2024 |
```

| Jim Madge <br>([@JimMadge](https://github.com/JimMadge)) | Senior Research Software Engineer, The Alan Turing Institute | The Turing Way Infrastructure Maintainers Co-lead | 2022 - Present | - |
| Brigitta Sipőcz <br>([@bsipocz](https://github.com/bsipocz)) | Research Software Engineer, Caltech/IPAC -- NASA/IPAC Infrared Science Archive | The Turing Way Infrastructure Maintainers Co-lead | 2022 - Present | - |
| Johanna Bayer <br>([@likeajumprope](https://book.the-turing-way.org/afterword/contributors-record#likeajumprope)) | Post Doctoral Fellow (CNS) | Infrastructure WG member | 2023 - Present | GSOC Co-Mentor 2023 |
| Carlos Martinez-Ortiz <br>([@c-martinez](https://github.com/c-martinez)) | RSE community manager, Netherlands eScience Center | Book Dash WG member| Feb 2020 - Present | - |
| Susana Roman Garcia <br>([@Susana465](https://book.the-turing-way.org/afterword/contributors-record#susana-roman-garcia)) | PhD researcher, University of Edinburgh | Book Dash WG member | 2024 - Present | - |

## Representation at Constitution Level Steering Committee

> The "Constitutional-level" participation in Steering Committee by community leaders allow development, approval and establishment of Working Group and project-related norms that affect the whole of the project, community and their sustainability.

Reviewer: ```suggestion
"Constitutional-level" participation in the Steering Committee by community leaders allow the development, approval and establishment of Working Group and project-related norms that affect the entire project, community and their sustainability.
```

Please read [Ways of Working document](https://github.com/the-turing-way/the-turing-way/blob/main/ways_of_working.md) for community structure, commitments and resources used by members at maintenance level, which also apply to constitution level.
Specific governance documentation and processes for this level will be developed with the founding members of the Steering Committee in 2025.

Chairs or a designated representative of the following **Working Groups** (WGs) join the Steering Committee at this level:

- **Community Management WG**
- **Infrastructure WG**
- **Book Dash WG**
- **Accessibility WG**
- **Translation and Localisation WG**

Leads or designated representatives of the **Delivery Teams** for the following projects within _The Turing Way_ also serve on the Steering Committee for accountability:

> - **_The Turing Way_ Practitioners Hub**, a Turing-Way project under the Innovate UK BridgeAI that engages with industries in sharing and adopting best practices for data science and AI.

Reviewer 2: ```suggestion
- **_The Turing Way_ Practitioners Hub**, a Turing Way project under the Innovate UK BridgeAI initiative that engages with industries in sharing and adopting best practices for data science and AI.
```

- **Software Sustainability Institute Fellows Projects**, that are defined to be conducted in collaboration with _The Turing Way_ project delivery or community members.

> - **JupyterHub -_The Turing Way_ Project**: [Arielle to add details]

Reviewer 2: ```suggestion
- **JupyterHub -_The Turing Way_ Project**: A CZI-funded project in collaboration with Organizational Mycology to reinvigorate community practices within JupyterHub and share generalisable findings with the wide community via _The Turing Way_.
```

| 📋    | Event Organizers                                                            |
| 💵    | Financial Support                                                           |
| 🔍    | Funding/Grant Finders                                                       |
| 🤔    | Ideas & Planning                                                            |
| 🚇    | Infrastructure (Hosting, Build-Tools, etc)                                  |
| 🔌    | Plugin/utility libraries                                                    |
| 👀    | Reviewed Pull Requests                                                      |
| 🔧    | Tools                                                                       |
| 🌍    | Translation                                                                 |
| ⚠️     | Tests                                                                       |
| ✅    | Tutorials                                                                   |
| 📢    | Talks                                                                       |
| 📹    | Videos                                                                      |

> If you have contributed in any of the ways above then please ask one of the Project Delivery team members to use the [all contributors bot :robot:](https://allcontributors.org/docs/en/bot/overview) to add you to the list.

Author: Anyone can add!
```suggestion
If you have contributed in any of the ways above then please ask one of the collaborators in *The Turing Way* to use the [all contributors bot :robot:](https://allcontributors.org/docs/en/bot/overview) to add you to the list.
```
